tests/routers/test_files.py

```
@@ -382,6 +382,21 @@ def test_thaw_file_success(
     mock_thaw_file.assert_called_once()
 
 
+def test_thaw_file_already_in_progress(
+    authenticated_client: TestClient, file_inventory_factory
+):
+    """Second thaw attempt should be blocked while the file is already migrating."""
+    cold_file = file_inventory_factory(
+        "/tmp/cold_in_progress.txt",
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp/cold_in_progress.txt"

(S108)